### DISCUSSION TITLE
Bad GitHub Account Management

### DISCUSSION THREAD

**Discussion by ihacksemant** [community/community#133042] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

it all started on May 28, 2024, I wanted to try transparent vscode, so I installed vscondium and other extentions on my vscode application, but they didn't work, so I just reinstalled the whole thing back to normal,

the next I tried to make a pr from my main account (s3mant) to a public repo (is-a.dev to be specific) and saw i couldn't create any prs not even clone repositories now

i searched online for the possible reasons and found out my account was flagged, no other info than that

i created a reinstatement ticket (id = https://support.github.com/ticket/personal/0/2837516) and waited and waited for 2 weeks and I got no response, then I made another ticket from this account mentioning the issue and then I received reply from a guy named Geoffrey, he said "make ticket from original account" bruh are you serious , I didn't get any response from that account that's why I made this, and then I replied to him again and got no response till now 

its almost 2 months and I've got no response from GitHub, this is causing me alot of mental pain, all my projects literally just vanished, no one can see my account nor can I contribute

please @github respond to me

**Comment by Budalebah** [comment]
is it fixed?
